docs/mutations.md

```diff
@@ -161,6 +161,11 @@ Do you have a suggestion for a (new) mutator? Feel free to create an [issue](htt
 | `$"foo {bar}"` | `$""` |
 | `@"foo"` | `@""` |
 | `string.Empty` | `"Stryker was here!"` |
+| `string.IsNullOrEmpty(x)` | `x != null` |
+| `string.IsNullOrEmpty(x)` | `x != ""` |
+| `string.IsNullOrWhiteSpace(x)` | `x != null` |
+| `string.IsNullOrWhiteSpace(x)` | `x != ""` |
+| `string.IsNullOrWhiteSpace(x)` | `x.Trim().Length != 0` |
```

Comment:
TBH That looks a bit off here, what's the justification for having it?

Reply:
Do you mean the `x.Trim().Length` mutation? This is check that the whitespace-only string is covered by unit tests. `Trim()` will return the zero-length string for the whitespace-only input.

This mutation also can be rewritten as `x.Trim() != ""` or using regexp.

Reply:
Right. I think the latter-most form is clearer.

Reply:
Rewritten using regexp

Reply:
Mhm not sure I agree @psfinaki , `x.Trim() != ""` is probably way more clear than regex, to most people.

Reply:
Damn I actually meant `x.Trim() != ""`🤦Probably my eyes naturally filter out anything about regex, anyway sorry for the confusioin.

Reply:
Refactored with `x.Trim()` )